gobblin-kubernetes/gobblin-service/azure-cluster/ingress.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: extensions/v1beta1
```

Comment:
why do we need nginx? For load balancing? Let's add some comments.

Reply:
Ah we need nginx as the ingress, because Azure AKS doesn't have a built in ingress like minikube does. I'll add some comments.